composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "gusdecool/cryptography",
-    "version": "1.0.0",
+    "version": "1.1.0",
```

Comment:
actually let's remove the `version` tag as in composer we can just use the git tag 👍